tests/test_cli.py

```diff
@@ -81,7 +81,7 @@ def test_no_traces(store_data, stdout, stderr):
     store, db_file = store_data
     with mock.patch.dict(os.environ, {DefaultConfig.DB_PATH_VAR: db_file.name}):
         ret = cli.main(['stub', func.__module__], stdout, stderr)
-    assert stderr.getvalue() == "No traces found\n"
+    assert stderr.getvalue().startswith("No traces found")
```

Comment:
Shouldn't we be able to use `func.__module__` again here and assert on the full error message?